src/components/AddContainerDialogComponent.vue

```diff
@@ -1,28 +1,42 @@
 <template>
-  <Dialog v-model:visible="visible" :header="$t('c_POSCreate.add container')">
+  <Dialog v-model:visible="visible"
+          :header="$t('c_POSCreate.add container')"
+          :draggable="false"
+          modal
+          @update:visible="closeDialog"
```

Comment:
Maybe use sudosos-pos-vue-3 utils dialogUtil to allow for click outside of dialog to close. I will most likely also move it to frontend-common.

Reply:
Yeah I was wondering already how to implement this, great you already built a utility for it. Fixed in the latest commit.